examples/tutorial/spc_to_dense_test.py

```diff
@@ -0,0 +1,50 @@
+import torch
```

Comment:
example shouldn't have the prefix "test", as an example should go beyond the scope of testing.